src/ServiceControl.Monitoring/HostApplicationBuilderExtensions.cs

```diff
@@ -87,6 +88,21 @@ static void ConfigureEndpoint(EndpointConfiguration config, Func<ICriticalErrorC
 
         transportCustomization.CustomizeMonitoringEndpoint(config, transportSettings);
 
+        var serviceControlThroughputDataQueue = settings.ServiceControlThroughputDataQueue;
+        if (!string.IsNullOrWhiteSpace(serviceControlThroughputDataQueue))
+        {
+            if (serviceControlThroughputDataQueue.IndexOf("@") >= 0)
```

Comment:
This is my lack of knowledge, but can we rely on all transport to use `@`?

Reply:
We set the queue name, so it's only if the user changes it.
It was code copied from here: https://github.com/Particular/ServiceControl/blob/monitoring-throughput-feature/src/ServiceControl.Audit/Infrastructure/NServiceBusFactory.cs#L36